Attempt 1- Please review

kapish5467 opened the pull request.

Gupta-Keshav-001378461 reviewed (commented):
Good Job, Kapish!! We will be using the Array Reader API for problem3. It will be covered in class if it's not clear then you can always DM me on slack.

Gupta-Keshav-001378461 commented:
Good Job, Kapish!! We will be using the Array Reader API for problem3. It will be covered in class if it's not clear then you can always DM me on slack.

kapish5467 commented:
> 
> 
> Good Job, Kapish!! We will be using the Array Reader API for problem3. It will be covered in class if it's not clear then you can always DM me on slack.

I have added a new code for Problem 3. Please feel free to add for future use ,as Leetcode non subscriber can not see the question. I have implemented the Arrayreader class.

Lohith-Aswathappa commented on Problem 3.java:
```diff
@@ -0,0 +1,73 @@
+//Step 3 if target<arr[i] set low and high variable and do a binary seach from low to high.
+
+//I am not able to build the Step 2 properly.
+
```
Well done, Kapish!
Include time and space complexity for this problem

Lohith-Aswathappa reviewed (commented):
well done, Kapish!
Check inline comments
